contrib/kube-prometheus/README.md

```diff
@@ -117,11 +130,7 @@ jsonnet -J vendor -m manifests "${1-example.jsonnet}" | xargs -I{} sh -c 'cat {}
 
```

Comment:
Shouldn't this bash script be changed to -m manifest "example.jsonnet" - to fit with above ?

Reply:
Great.. running jssonnet on that example.jsonnet gave me a manifests folder looking much like whats in the manifests folder in the project.
Now on to the customization part.. You start with saying there's a hidden _config folder.. But I can find no such folder? What should I have done - to get that ?

Reply:
I assume I should do something - change example.jsonnet to include something from f.ex. https://github.com/brancz/kubernetes-grafana - to modify the generated manifest for grafana..
and then just apply all manifests again - to adjust my running config in kubernetes?
What is the recommended approach to work with this in production..
1. modify "my-config.jsonnet", 
2. use jsonnet to compile manifests 
3. kubectl apply -f manifests/*
?

Reply:
I would f.ex. like to change which namespaces this config should apply to/monitor (per https://github.com/coreos/prometheus-operator/issues/1696 ) - I think that should be mentioned (that i by default ONLY monitores 'monitoring' namespace and how to adjust example.jsonnet to monitor more namespaces.

My plan is to build a my-prodconfig.jsonnet and a my-testconfig.jsonnet and then list the namespaces they each apply to.. (ie. a list of test namespaces in testconfig and prod namespaces in prodconfig).. so I can test changes, by applying them in testconfig first and kubectl apply that first..
does that sound reasonable? If so - IMHO it should be added to this doc as "how to use it"

Reply:
"an entirely seperate prometheus server".. so if I want to monitor services/pods in "test" namespace.. that config should not "just" be added to my-config.jsonnet (config section) ?

Reply:
what is your definition of "entirely seperate prometheus server".. ? Something setup using just core "prometheus- operator" I assume? and that won't conflict with using kube-prometheus?.. The common way - is to then have 1 prometheus setup for monitoring kubernetes ONLY. And then one f.ex. for "test" namespaces and one for "prod namespaces" - monitoring services ON kubernetes setup (which is NOT kubernetes itself) ? It would be nice with a small "recommended approach" - and pointer to what to follow - AFTER having setup kube-prometheus.. (I assume thats just the core repo of prometheus-operator :)

Reply:
Is there any way to parse check a jsonnet file ?
See https://github.com/coreos/prometheus-operator/issues/1731 - for an issue I got when trying to put your examples together.. the compile error is not exactly helping me to identify the issue :(